samples/JSON/VCT.yy

```diff
@@ -0,0 +1,13 @@
+{
+  "$GMScript":"v1",
```

Comment:
Maybe this `$GMScript` is a better one to match on given that it sounds pretty exclusive to GM? And looks like its a fixed line too.

Not sure if this is true for all newer files.. Haven't touched GM in a while :(

Reply:
Hum, based on [this search](https://github.com/search?q=path%3A*.yy+%2F%5CA%28%3F%3A.*%28%3F%3A%5Cr%3F%5Cn%7C%5Cr%29%29%7B0%2C%7D%5Cs*%5C%22%28%5C%24GMScript%29%5C%22%5C%3A%5Cs*%5C%22v%2F++NOT+%2F%5CA%28%3F%3A.*%28%3F%3A%5Cr%3F%5Cn%7C%5Cr%29%29%7B0%2C%7D%5Cs*%5C%22%28modelName%7CresourceType%29%5C%22%5C%3A%5Cs*%5C%22GM%2F+&type=code), there is no files that has `$GMScript` and not `modelName` or `resourceType`, so adding `$GMScript` to the matching rules wouldn't help match more files at least.

Reply:
$GMScript and things like that seem to appear in the first 2 lines from what I've seen - `modelName` and `resourceType` might be all the way at the bottom (or in the middle if we're talking "room" files) :thinking: